### PR TITLE
Adds german language to Genisys and it's installer

### PR DESCRIPTION
**Pull request by SleepSpace9**
### Description
This PR adds german language to Genisys and the setup wizard (ToDo: genisys_deu.yml).

### Reason to modify
Language was missing and a lot of people could like it.

### Tests & Reviews
Tested the setup wizard and also some output on console and in-game.

### CODE REVIEW

**Comment by SleepSpace9** [comment]
As I already asked on Gitter, I'm not sure about the licence thing - the Installer claimes "LGPL" and the language file in locale says "GPL License version 3 and later". ❔

**Comment by ishitatsuyuki** [comment]
Inviting @robske110 for a review.

**Comment by robske110** [comment]
Full review incoming. (And it should be GPL, someone messed the installer up *cough* PeratX *cough*)

**Comment by dktapps** [comment]
@robske110 nobody messed it up, it simply wasn't changed from the original. The original PocketMine licence is LGPL.

**Comment by erkam2002** [comment]
Good

File: src/pocketmine/lang/Installer/deu.ini [hunk 1]
```diff
@@ -0,0 +1,51 @@
+language_has_been_selected = Deutsch wurde erfolgreich als Sprache ausgewählt.
+skip_installer = Möchtest du den Einrichtungsassistenten überspringen?
+
+welcome_to_pocketmine = Willkommen bei Genisys!\nBevor es mit der Einrichtung deines neuen Servers losgehen kann, musst du die Lizenz akzeptieren.\nGenisys ist unter der LGPL Lizenz lizenziert, welche du in der Datei LICENCE in diesem Ordner nachlesen kannst.
```
Comment (SleepSpace9) [inline src/pocketmine/lang/Installer/deu.ini]: Stimmt, man sollte eben doch nicht zu sehr am Original-Text kleben -> shit in = shit out :smile:
"...that you can read opening the LICENSE file on this folder."
Es würde sicherlich auch reichen "...welche du in der Datei LICENCE nachlesen kannst."

File: src/pocketmine/lang/Installer/deu.ini [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+accept_license = Akzeptierst du die Lizenz?
+you_have_to_accept_the_license = Um Genisys weiterverwenden zu können, musst du der LGPL Lizenz zustimmen.
```
Comment (SleepSpace9) [inline src/pocketmine/lang/Installer/deu.ini]: Okay, if I have understood the Genisys members correctly then we should change it to GPL v3 and later. This also means that we have to do it in all the other language files of the Installer too.

File: src/pocketmine/lang/Installer/deu.ini [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+setting_up_server_now = Du bist nun dabei, deinen Server einzurichten.
+default_values_info = Drücke einfach Enter, um die Standardwerte zu übernehmen.
+server_properties = Du kannst sie auch später in der Datei server.properties ändern.
+
+name_your_server = Gib deinem Server einen Namen.
+port_warning = Ändere nicht den Wert für den Standart-Port, falls dies dein erster Server ist.
+server_port = Server-Port
+invalid_port = Ungültiger Server-Port
+online_mode_info = Im Online-Modus wird der User gezwungen, sich mit seinem XBOX-Account zu authentifizieren.\nEs wird die Aktivierung des Online-Modus empfohlen, falls der Server für externe Spieler geöffnet ist.
```
Comment (SleepSpace9) [inline src/pocketmine/lang/Installer/deu.ini]: "Spieler" finde ich gut

File: src/pocketmine/lang/Installer/deu.ini [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+spawn_protection = Spawnschutz aktivieren?
+level_name = Bestimme den Namen der Welt
+level_type = Lege fest, welchen Welttyp der Server verwenden soll
+invalid_level_type = Ungültiger Welttyp
+announce_player_achievements = Soll benachrichtigt werden, wenn der Spieler einen Erfolg erzielt?
+
+op_info = Ein OP ist ein Spieler, der Admin ist. OPs können mehr Befehle ausführen als normale Spieler.
+op_who = OP Spielername (z.B. dein Spielername)
+op_warning = Du kannst später weitere OP Spieler hinzufügen, indem du eingibst /op <Spielername>
+whitelist_info = Mit Aktivierung der Whitelist dürfen nur noch Spieler joinen, die hierin enthalten sind.
```
Comment (SleepSpace9) [inline src/pocketmine/lang/Installer/deu.ini]: "darin" is my favourite, I don't like repeating words to often

File: src/pocketmine/lang/Installer/deu.ini [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+query_warning2 = Wenn du es deaktivierst, kannst du keine Serverlisten nutzen.
+query_disable = Möchtest du Query deaktivieren?
+rcon_info = RCON ist ein Protokoll, über welches man sich aus der Ferne mit einem Passwort zur Serverkonsole verbinden kann.
+rcon_enable = Möchtest du RCON aktivieren?
+rcon_password = RCON Passwort (du kannst es später ändern)
+usage_info = Die anonymen Nutzungsdaten ermöglichen uns, eine globale Statistik für Pocketmine-MP und dessen Plugins zu generieren. Du kannst sie dir auf stats.pocketmine.net ansehen.
+usage_disable = Möchtest du die anonymen Nutzungsdaten deaktivieren?
+ip_get = Deine interne und externe IP-Adresse wird ermittelt
+ip_warning = Deine externe IP ist {{EXTERNAL_IP}}. Möglicherweise muss eine Port-Weiterleitung zu deiner internen IP {{INTERNAL_IP}} eingerichtet werden.
+ip_confirm = Stelle sicher, dass du das geprüft hast. Falls du eine Weiterleitung benötigst, aber nicht einrichtest, wird kein externer Spieler joinen können. [Drücke Enter]
```
Comment (SleepSpace9) [inline src/pocketmine/lang/Installer/deu.ini]: Das Nomen wäre hier stellvertretend "das" und bezieht sich auf den Satz davor, also für mich klar verständlich. Aber ich bin offen für Gegenvorschläge.

File: src/pocketmine/lang/Installer/deu.ini [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+server_ram = Server RAM in MB
+gamemode_info = Wähle zwischen Kreativmodus (1) oder Überlebensmodus (0)
+default_gamemode = Standard Spielmodus
+max_players = Max. Online-Spieler
+spawn_protection_info = Der Spawnschutz (spawn protection) verhindert das Setzen oder Abbauen von Blöcken im Spawn-Bereich, außer für OPs.
+spawn_protection = Spawnschutz aktivieren?
+level_name = Bestimme den Namen der Welt
+level_type = Lege fest, welchen Welttyp der Server verwenden soll
+invalid_level_type = Ungültiger Welttyp
+announce_player_achievements = Soll benachrichtigt werden, wenn der Spieler einen Erfolg erzielt?
```
Comment (SleepSpace9) [inline src/pocketmine/lang/Installer/deu.ini]: Hmm... würde 2x "Spieler" bedeuten - Sollen die **Spieler** benachrichtigt werden, wenn der **Spieler** einen Erfolg erzielt?"

Vielleicht besser: "Sollen die Spieler benachrichtigt werden, wenn ein Erfolg erzielt wird?"

File: src/pocketmine/lang/Installer/deu.ini (hunk 5 above)
Comment (robske110) [inline src/pocketmine/lang/Installer/deu.ini]: Oh, da habe ich wohl nicht den vorherigen Satz gelsen. Dann macht es mehr sinn.

File: src/pocketmine/lang/Installer/deu.ini (hunk 6 above)
Comment (robske110) [inline src/pocketmine/lang/Installer/deu.ini]: Jop, deine Verbesserung ist gut :)

File: src/pocketmine/lang/Installer/deu.ini (hunk 2 above)
Comment (robske110) [inline src/pocketmine/lang/Installer/deu.ini]: Yes, shortening is a good approach.

File: src/pocketmine/lang/Installer/deu.ini (hunk 1 above)
Comment (robske110) [inline src/pocketmine/lang/Installer/deu.ini]: Yes, it looks like the original text also has that proble.

**Comment by SleepSpace9** [comment]
Okay, I think the PR is ready! I rebased and squashed it (was an adventure xD).

**Comment by dktapps** [comment]
@robske110 any further objections?

**Comment by robske110** [comment]
Nope, ready to merge

**Pull request merged**